easybuild/easyconfigs/m/mii/mii-1.1.0.eb

```diff
@@ -34,13 +34,19 @@ sanity_check_paths = {
 sanity_check_commands = [('mii', 'version')]
 
 modluafooter = """
+local index_root = "/cvmfs/soft.computecanada.ca/custom/mii"
+
 setenv("MII", pathJoin(root, "bin/mii"))
-setenv("MII_INDEX_FILE", pathJoin(root, "data/index"))
+setenv("MII_INDEX_FILE", pathJoin(index_root, "data/index"))
 
 local hook_name = "command_not_found_handle"
 local hook_def = "source " .. pathJoin(root, "share/mii/init/common") .. " $@"
 
--- Define hooks for both bash and zsh
+-- Correct the hook name if shell is zsh
+if (myShellName() == "zsh") then
+    hook_name = hook_name .. 'r'
+end
+
+-- Define command not found hook
```

Comment:
Should we give an error if the shell is not zsh or bash ?

Reply:
Something like
```
if (myShellName() == "zsh" or myShellName() == "bash") then
    set_shell_function(hook_name, hook_def, "")
else
    if (mode() == "load") then
        LmodError("This module can not be used with shells other than bash or zsh")
    end
end

Reply:
Mii can work with other shells, but the shell integration only supports `bash` and `zsh`. We could register the command not found hook only if the shell is `bash` or `zsh`

Reply:
For example, `mii search blastn` can be used even with `tcsh` to find which modules provide the `blastn` command.

Reply:
good point.